ps_facetedsearch.php

```diff
@@ -267,6 +269,36 @@ public function indexAttributes($idProduct = null)
         );
     }
 
+    /*
+     * Generate data for product features
+     *
+     * @return boolean
+     */
+    public function indexFeatures()
+    {
+        return $this->getDatabase()->execute(
+            'INSERT INTO `' . _DB_PREFIX_ . 'layered_indexable_feature` ' .
+            'SELECT id_feature, 1 FROM `' . _DB_PREFIX_ . 'feature` ' .
+            'WHERE id_feature NOT IN (SELECT id_feature FROM ' .
+            '`' . _DB_PREFIX_ . 'layered_indexable_feature`)'
```

Comment:
Do we really need the backticks ? This would make it harder to use the module with PostgreSQL

Reply:
Since we didn't plan to make it work with pgsql, using backticks are still the good way for mysql :/